api/import_export/export.py

```diff
@@ -83,6 +85,18 @@ def export_organisation(organisation_id: int) -> typing.List[dict]:
     )
 
 
+def export_metadata(organisation_id: int) -> typing.List[dict]:
+    return _export_entities(
+        _EntityExportConfig(MetadataField, Q(organisation__id=organisation_id)),
+        _EntityExportConfig(
+            MetadataModelField, Q(field__organisation__id=organisation_id)
+        ),
+        _EntityExportConfig(
+            Metadata, Q(model_field__field__organisation__id=organisation_id)
```

Comment:
You mean with MetadataModelField ?

Reply:
Yes, sorry, I think I put the comment in the wrong place.

Reply:
It does have UUID, though?

Reply:
But it's not a traditional foreign key, right? So I'm not sure how it will link to the relevant entities when we export it (I can't quite remember how the export works tbh!)